cmd/runtimetest/main.go

```diff
@@ -370,6 +371,62 @@ func validateMountsExist(spec *rspec.Spec) error {
 	return nil
 }
 
+func getActualCgroupPath() (string, error) {
+	f, err := os.Open("/proc/self/cgroup")
+	if err != nil {
+		return "", err
+	}
+	defer f.Close()
+
+	cgroupPath := ""
+	s := bufio.NewScanner(f)
+	for s.Scan() {
+		if err := s.Err(); err != nil {
+			return "", err
+		}
+
+		text := s.Text()
+		parts := strings.Split(text, ":")
+		if cgroupPath == parts[2] && cgroupPath != "" {
+			continue
+		} else if cgroupPath == "" && parts[2] != "" {
+			cgroupPath = parts[2]
+		} else {
+			return "", fmt.Errorf("error path with cgroup controllers")
```

Comment:
This will break for runtimes that ignore unneeded cgroups (which opencontainers/runtime-spec#493 allows).  See [here][1] and [here][2] for more.

[1]: https://github.com/opencontainers/ocitools/pull/93#discussion_r73002225
[2]: https://github.com/opencontainers/ocitools/pull/93#discussion_r73188918